components/scream/src/share/util/scream_column_ops.hpp

```diff
@@ -607,6 +636,68 @@ class ColumnOps {
     }
   }
 
+  // +--------------------------------------------+
+  // |        Kokkos::parallel_X wrappers         |
+  // +--------------------------------------------+
+
+  // Runs the input lambda with a TeamThreadRange parallel for over [0,count) range
+  template<typename Lambda>
+  KOKKOS_INLINE_FUNCTION
+  static void team_parallel_for (const TeamMember& team,
```

Comment:
Question: why do we use the `static` keyword here? Is it to prevent this stuff from being used elsewhere? Is it needed in order to inline the function? If the former, we could place these functions into an anonymous namespace.

Not a big deal--just wondering about the reasoning.

Reply:
In principle, we don't care about static here, since these methods are only used from within the class, not from outside. However, you cannot call a non-static method from a static one, and the main methods of the class are static.

Reply:
Ahhhh, okay. I missed that these appeared in a class declaration(!). Wrong `static`. Thanks  Luca.

Reply:
I wish C++ had not overloaded the static keyword...